### DISCUSSION TITLE
add to flipperzero-plugins

### DISCUSSION THREAD

**Discussion by fennectech** [akopachov/flipper-zero_authenticator#41] [Ideas]
https://github.com/akopachov/flipper-zero_authenticator/issues/new  you can even use a subrepo to get your stuff updated in there automagicly

**Comment by akopachov** [comment]
Would you mind providing a link to that catalog?

**Comment by fennectech** [comment]
https://github.com/djsime1/awesome-flipperzero   Here’s the awesome flipper hub.   could list it here.   as well as this other package that includes a bunch of precompiled apps https://github.com/vmeurisse/flipperzero-plugins

**Comment by akopachov** [comment]
For https://github.com/djsime1/awesome-flipperzero started new discussion [here](https://github.com/djsime1/awesome-flipperzero/discussions/72)
